internal/commands/fetch.go

```diff
@@ -163,6 +163,10 @@ nextRelease:
 				intersection = append(intersection, rel)
 				missing = append(missing[:j], missing[j+1:]...)
 				continue nextRelease
+			} else if rel.Lock.Name == lock.Name && rel.Lock.Version == lock.Version {
+				fmt.Printf("release: [ %s ] sha mismatch: [ %s ]\n", lock.Name, rel.Lock.SHA1)
```

Comment:
Thanks for doing this @pnikonowicz, could you please just add the word local to both messages so the message is clear that we are printing the information for local releases, something like: `fmt.Printf("Local release: [ %s ] sha mismatch: [ %s ]\n", lock.Name, rel.Lock.SHA1)`